code/modules/uplink/uplink_items/device_tools.dm

```diff
@@ -82,6 +83,7 @@
 			Comes with 4 charges, recharges randomly. Warranty null and void if exposed to an electromagnetic pulse."
 	item = /obj/item/storage/box/syndie_kit/syndicate_teleporter
 	cost = 8
+*/
```

Comment:
```suggestion
*/ //END SKYRAT EDIT
```
IDK if this is a mandatory thing but most block changes have the end comment

Reply:
It tends to be, yeah.